modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -62,11 +62,18 @@ When using shared storage, review your security settings to prevent outside acce
 $ oc get pod -n openshift-image-registry
 ----
 +
+.Example output
+[source,yaml]
+----
+NAME                                               READY   STATUS      RESTARTS   AGE
+node-ca-7tfjt                                      1/1     Running     0          95d
+node-ca-8f82b                                      1/1     Running     0          95d
```

Comment:
Better to use `oc get pod -n openshift-image-registry -l docker-registry=default`

```
oc get pod -n openshift-image-registry -l docker-registry=default
No resources found in openshift-image-registry namespace.
```

Reply:
@xiuwang Just updated, Thank you!

Reply:
@kelbrown20 Hi, I didn't make it clear,
when use `oc get pod -n openshift-image-registry -l docker-registry=default` should return `No resources found in openshift-image-registry namespace.` when no registry pod set. But not return the node-ca pods list.

Reply:
Oh I see, my apologies for the confusion! Should the `-l docker-registry=default` still be in the documented command?